hardware/raspberrypi/booteeprom.md

```diff
@@ -6,104 +6,148 @@ The Raspberry Pi 4 has an SPI-attached EEPROM (4MBits/512KB), which contains cod
-To view the configuration file used by the bootloader at boot time run `rpi-eeprom-config` or `vcgencmd bootloader_config`.
+* `default` - Updated for new hardware support, critical bug fixes and periodic update for new features that have been tested via the `latest` release.
+* `latest` - Updated when new features have been successfully beta tested.
+* `beta` - New or experimental features are tested here first.
 
-### Reading the configuration file from an EEPROM image
-To read the configuration file from an EEPROM image file:
-```
-rpi-eeprom-config pieeprom.bin
-```
+Since the release status string is just a subdirectory name then it's possible to create your own release streams e.g. a pinned release or custom network boot configuration.
 
-### Editing the current bootloader configuration
-The following command loads the current EEPROM configuration into a text editor. When the editor is closed, `rpi-eeprom-config` applies the updated configuration to latest available EEPROM release and uses `rpi-eeprom-update` to schedule an update when the system is rebooted:
+N.B. `default` and `latest` are symbolic links to the older release names of `critical` and `stable`.
+
+### Changing the bootloader release
 
+You can change which release stream is to be used during an update by editing the `/etc/default/rpi-eeprom-update` file and changing the `FIRMWARE_RELEASE_STATUS` entry to the appropriate stream.
+
+## Low level commands
+
+## rpi-eeprom-update
+Raspberry Pi OS uses the `rpi-eeprom-update` script to implement an [automatic update](#automaticupdates) service. The script can also be run interactively or wrapped to create a custom bootloader update service.
+
+Reading the current EEPROM version:  
 ```
-sudo -E rpi-eeprom-config --edit
-sudo reboot
+vcgencmd bootloader_version
 ```
-If the updated configuration file is identical or empty then no changes are made.
 
-The editor is selected by the `EDITOR` environment variable.
+Check if an update is available:  
+```
+sudo rpi-eeprom-update
+```
 
-### Applying a saved configuration file
-The following command applies `boot.conf` to the latest available EEPROM image and uses `rpi-eeprom-update` to schedule an update when the system is rebooted.
+Install the update:  
 ```
-sudo rpi-eeprom-config --apply boot.conf
+sudo rpi-eeprom-update -a
 sudo reboot
 ```
 
-### Low level commands
+Cancel the pending update:  
+```
+sudo rpi-eeprom-update -r
+```
 
-#### Updating the configuration in an EEPROM image
-The following command reads `pieeprom.bin` and replaces the configuration file with the contents of `boot.conf`. The result is written to `new.bin`
+Installing a specific bootloader EEPROM image:  
 ```
-rpi-eeprom-config --config boot.conf --out new.bin pieeprom.bin
+sudo rpi-eeprom-update -d -f pieeprom.bin
 ```
+The `-d` flag instructs `rpi-eeprom-update` to use the configuration in specified image file instead of automatically migrating the current configuration.
```

Comment:
`in specified image file` -> `in the specified image file`

Reply:
done